plat/qti/common/inc/aarch64/plat_macros.S

```diff
@@ -0,0 +1,110 @@
+/********************************************************************
+ * Copyright (c) 2018 Qualcomm Technologies, Inc.
+ * All Rights Reserved.
+ * Confidential and Proprietary - Qualcomm Technologies, Inc.
+ *********************************************************************/
+
+/*
+ * Copyright (c) 2015-2017, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
```

Comment:
I'm leaving the comment here but it applies to the other files. You can't say "confidential and proprietary" and then use BSD license. If you want to use Qualcomm in the copyright notice, that's fine, but this is contradictory.

Reply:
Hi @antonio-nino-diaz-arm,
We will talk to our legal team regarding dual licensing. 
We need to get some scan done for every release, so can you please do a complete review for this request so that we can provide you all update together.
Thanks for your help

Reply:
@sgorecha,  The "confidential and proprietary" notice prevents us from giving a IP-Review +1. The further inspection of code can only be done after IP-Review has passed.